README.md

```diff
@@ -1,2 +1,185 @@
+
+# 📖 Hello_microverse <a name="about-project"></a>
+
+> Describe your project in 1 or 2 sentences.
+
+**[Hello Microverse]** is an initiation project to show micronauts on how to build future projects. It uses linters to check for errors and it contains an html file that displays an header of "Hello Microverse!". It also contains a css file thaat styles the header with a color of blue. It also contains a gitignore file that contains folders and files that are to be ignoreed by git.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+> Describe the tech stack and include only the relevant sections that apply to your project.
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://reactjs.org/">HTML & CSS</a></li>
+  </ul>
+</details>
+
+<!-- <details>
+  <summary>Server</summary>
+  <ul>
+    <li><a href="https://expressjs.com/">Express.js</a></li>
+  </ul>
+</details> -->
+
+<!-- <details>
+<summary>Database</summary>
+  <ul>
+    <li><a href="https://www.postgresql.org/">PostgreSQL</a></li>
+  </ul>
+</details> -->
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+> Describe between 1-3 key features of the application.
+
+- **Displays an text of Hello Microverse**
+- **Styled the text with a color of blue**
+- **Ignored node_modules and test.md**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+> Describe how a new developer could make use of your project.
+
+To get a local copy up and running, follow these steps.
+
+- **Click on the the code button on this repository to copy the links**
+- **copy either the SSH link or the HTTPS link**
+- **Open the terminal in you text editor**
+- **Navigate using the command line command to a prefered directory to save the project folder**
+- **Type in "git clone <clone-link>"**
+- **Then open the index.html file**
+- **Right click onn the interface and click on "open in browser"**
+
+
+
+### Prerequisites
+
+In order to run this project you need:
+
+VScode/Bracket/Sublime/Atom
+A Browser
+<!--
+Example command:
+
+```sh
+ gem install rails
+```
+ -->
+
+### Setup
+
+Clone this repository to your desired folder:
+
+
+```sh
+  cd my-folder
+  git clone git@github.com:myaccount/my-project.git
+```
+
+
+
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+> Mention all of the collaborators of this project.
+
+👤 **Ridwan Alabi**
+
+- GitHub: [@githubhandle](https://github.com/Ridalord)
+- Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
+- LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle)
+
+👤 **Author2**
+
+- GitHub: [@githubhandle](https://github.com/githubhandle)
+- Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/ridwan-alabi-069317246/)
```

Comment:
- [ ] Kindly add the corrrect links to your linkedIn and Twitter as they currently do not lead to anywhere when clicked, It is important to provide working links as recruiters could easily reach you via these links. :+1: 
- [ ] Kindly remove all of the descriptions for author 2 since there is currently no second author for this project. It is good to make your readme file as professional as possible without adding text which can be done away with :+1: